api/Permissions.json

```diff
@@ -726,7 +726,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "16.5"
+              "version_added": "17"
```

Comment:
Appears to have been implemented via https://github.com/WebKit/WebKit/commit/0789169e66412b631fddd23369aa42a65592721a in WebKit [616.1.14](https://github.com/WebKit/WebKit/blob/0789169e66412b631fddd23369aa42a65592721a/Configurations/Version.xcconfig#L24-L26) before Safari [17](https://github.com/mdn/browser-compat-data/blob/63a7d3e9df4ea2566dab77ec8ec080be2ca2224f/browsers/safari.json#L272-L278).